qiskit_experiments/framework/base_experiment.py

```diff
@@ -192,8 +207,94 @@ def from_config(cls, config: Union[ExperimentConfig, Dict]) -> "BaseExperiment":
             ret.set_transpile_options(**config.transpile_options)
         if config.run_options:
             ret.set_run_options(**config.run_options)
+        if config.analysis:
+            ret.analysis = config.analysis.analysis()
         return ret
 
+    def save(self, service=None, backend=None):
+        """Save an experiment without running it."""
+        if not service:
+            raise QiskitError("A service must be provided to save the experiment.")
+        if not self.backend and not backend:
+            raise QiskitError("Backend must be set to save the experiment.")
+        exp_data = self._initialize_experiment_data(
+            service=service, backend=backend or self.backend
+        )
+        exp_data.save()
+        return exp_data
+
+    @classmethod
+    def load(
+        cls,
+        experiment_id: str,
+        service: Optional[IBMExperimentService] = None,
+        provider: Optional[Provider] = None,
```

Comment:
If `service` it not None, but `provider` is None, we get this warning: 

> /tmp/qiskit-experiments/qiskit_experiments/framework/base_experiment.py:286: UserWarning: Unable to retrieve backend.
  warnings.warn("Unable to retrieve backend.")

I think this should be fixed - either don't allow an empty provider or update the doc string to explain the behavior when `provider` is empty. Currently the doc string explicitly says that `IBMProvider` is required